data/json/items/melee.json

```diff
@@ -2283,5 +2283,41 @@
     "techniques": "RAPID",
     "qualities": [ [ "CUT", 1 ], [ "CUT_FINE", 1 ], [ "BUTCHER", 7 ] ],
     "flags": [ "STAB", "SHEATH_KNIFE" ]
+  },
+  {
+    "id": "2h_flail_steel",
+    "type": "GENERIC",
+    "category": "weapons",
+    "name": "war flail",
+    "description": "This is a stout pole with a large steel flanged mace head on a short chain attached to it, based on the peasant flail agricultural tool except now with a metal head and made to thresh people in metal armor rather than grain.",
+    "weight": 4550,
+    "volume": 15,
+    "price": 25000,
+    "to_hit": 0,
+    "bashing": 40,
+    "cutting": 0,
```

Comment:
Most two-handed flails I've seen have spikes or whatnot on a mace head, and since this seems to be a replica of such a medieval weapon, would you mind considering adding minor cutting damage (with some edits in the description that reflect the changes) ?

Reply:
> Most two-handed flails I've seen have spikes or whatnot on a mace head, and since this seems to be a replica of such a medieval weapon, would you mind considering adding minor cutting damage (with some edits in the description that reflect the changes) ?

the head of the wair flail I added is flanged, not spiked, I had this in mind when writing that:

![flail_two_handed_02_named_01](https://user-images.githubusercontent.com/29681506/59553729-924a5e80-8f6f-11e9-9e10-94b2ca389233.png)

I don't know if the hard corners would do cut damage but I guess not since they would be hard edges to transfer more of the impact to a single point.

Reply:
Yeah, seems fair.

Although, it also seems that both flat and spiky flails are called _Kettenmorgenstern_. Which is just a generalisation, I believe. I am not a medieval arms expert by any means, so I tried to check arts and books at wikimedia: most of the flat ones are seems like a peasant flails, and others are with spikes.

I just thought that making flat and spiky versions will result in almost identical weapons which is a little redundant.

Reply:
The idea is that to peasant flail can be auto-learned once you reach a certain level of skill, but the war one requires the book, so it might just be a straight upgrade without variation but it requires one to find a specific book which given the nature of RNG might not happen, also, flanged ones are cooler than spiked ones :P